### PR TITLE
Skill - Merciless

### PR DESCRIPTION
**Pull request by JesterWizard**
Guaranteed critical hit against poisoned foes

### CODE REVIEW

File: Wizardry/Core/BattleSys/Source/PreBattleCalc.c [hunk 1]
```diff
@@ -428,6 +428,16 @@ STATIC_DECLAR void PreBattlePostCalcSkills(struct BattleUnit * attacker, struct
             attacker->battleDefense = INT16_MAX;
     }
 #endif
+
+#if (defined(SID_Merciless) && (SID_Merciless < MAX_SKILL_NUM))
+    if (SkillTester(unit, SID_Merciless))
+    {
+        //Check if the defending unit has the poison status
+        if(GetUnitStatusIndex(&defender->unit) == UNIT_STATUS_POISON)
+            //If so, then set an arbitrary high value for crit to 'gurantee' it.
```
Comment (MokhaLeee) [inline Wizardry/Core/BattleSys/Source/PreBattleCalc.c:437]: add space after "//"

File: Wizardry/Core/BattleSys/Source/PreBattleCalc.c [hunk 2]
```diff
@@ -428,6 +428,16 @@ STATIC_DECLAR void PreBattlePostCalcSkills(struct BattleUnit * attacker, struct
             attacker->battleDefense = INT16_MAX;
     }
 #endif
+
+#if (defined(SID_Merciless) && (SID_Merciless < MAX_SKILL_NUM))
+    if (SkillTester(unit, SID_Merciless))
+    {
+        //Check if the defending unit has the poison status
+        if(GetUnitStatusIndex(&defender->unit) == UNIT_STATUS_POISON)
```
Comment (MokhaLeee) [inline Wizardry/Core/BattleSys/Source/PreBattleCalc.c:436]: add space after "if"

File: Wizardry/Core/BattleSys/Source/PreBattleCalc.c (hunk 1 above)
Comment (JesterWizard) [reply Wizardry/Core/BattleSys/Source/PreBattleCalc.c:437]: Done.

File: Wizardry/Core/BattleSys/Source/PreBattleCalc.c (hunk 2 above)
Comment (JesterWizard) [reply Wizardry/Core/BattleSys/Source/PreBattleCalc.c]: Done.

**Pull request merged**